discord/guild.py

```diff
@@ -735,6 +744,16 @@ def filesize_limit(self) -> int:
     def members(self) -> List[Member]:
         """List[:class:`Member`]: A list of members that belong to this guild."""
         return list(self._members.values())
+    
+    @property
+    def humans(self) -> List[Member]:
+        """List[:class:`Member`]: A list of humans that belong to this guild."""
+        return list(mem for mem in self._members.values() if not m.bot)
```

Comment:
Why not `self.members` though

Reply:
there is no difference between internal variable and public property
```suggestion
        return list(mem for mem in self.members if not m.bot)
```

Reply:
Sure, lemme fix that!